src/index.html

```diff
@@ -2,13 +2,21 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Snake</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Snake</h1>
+  <div class="container">
+    <div class="block block--1">1</div>
```

Comment:
```suggestion
    <div class="block">1</div>
```
Ты же не используешь эти классы

Reply:
после коммита заметил  сейчас что при разрешении 1200 у меня почему-то 5 и 6 блоки становятся по 600 px и все ломается, почему так ?